src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import store from './Components/Pages/Redux/ConfigureStore';
+import Main from './Components/Pages/Main';
+import Details from './Components/Pages/Details';
+
+function App() {
+  return (
+    <div className="App">
+      <Provider store={store}>
+        <Routes>
+          <Route path="/main" element={<Main />} />
+          <Route path="/detail/:id" element={<Details />} />
+        </Routes>
```

Comment:
- [ ] Add a main path `/` because when the users first starts your app it sees only a blank page

Reply:
ok fixed